src/config.yaml

```diff
@@ -1,28 +1,29 @@
-apiVersion: kubelet.config.k8s.io/v1beta1 
-kind: KubeletConfiguration 
-# kubelet in standalone 
-authentication: 
- anonymous: 
-  enabled: false 
- webhook: 
-  enabled: false 
-authorization: 
- mode: AlwaysAllow 
-# Define POD CIDR 
-podCIDR: 10.0.0.0/16 
-# static pod for stand-alone mode 
-staticPodPath: /etc/kubelet.d/ 
-cgroupDriver: systemd 
-# DPU management address 
-# address: <DPU IP> 
-# imageMinimumGCAge: 
-# imageGCHighThresholdPercent: 95 
-# imageGCLowThresholdPercent: 80 
-# maxPods: 3 
-# systemReserved: 
-#  cpu: "300m" 
-#  Memory: "300Mi" 
-featureGates:  
- TopologyManager: false 
- CPUManager: false 
- DevicePlugins: false 
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+containerRuntimeEndpoint: unix:///run/containerd/containerd.sock
+# kubelet in standalone
+authentication:
+ anonymous:
+  enabled: false
+ webhook:
+  enabled: false
+authorization:
+ mode: AlwaysAllow
+# Define POD CIDR
+podCIDR: 10.0.0.0/16
+# static pod for stand-alone mode
+staticPodPath: /etc/kubelet.d/
+cgroupDriver: systemd
+# DPU management address
+# address: <DPU IP>
+# imageMinimumGCAge:
+# imageGCHighThresholdPercent: 95
+# imageGCLowThresholdPercent: 80
+# maxPods: 3
+# systemReserved:
+#  cpu: "300m"
+#  Memory: "300Mi"
+#featureGates:
```

Comment:
Maybe remove the `featureGates` section also from comments?

Reply:
Basically I would say maybe we should kick out everything that's commented?..
What do you say @rollandf ?

Reply:
I am not sure about the rest and if it is there as a reference.
I would just remove the featureGates for now.

Reply:
Done :)